packages/app-desktop/integration-tests/util/mockNextShowMessageDialog.ts

```diff
@@ -0,0 +1,84 @@
+
+import type { ElectronApplication } from '@playwright/test';
+import { BrowserWindow, MessageBoxOptions } from 'electron';
+
+const mockNextShowMessageCall = (
```

Comment:
For reference, compare to [`playwright-fake-dialog`](https://github.com/wsw0108/playwright-fake-dialog/blob/master/index.js). (Using this library would likely still require re-implementing much of the below code).